feature/server.go

```diff
@@ -121,13 +125,27 @@ func (s *server) SetFeature(ctx context.Context, req *featurepb.SetFeatureReques
 		}
 	}
 
+	defer s.notifyChange()
```

Comment:
In the future we should compute whether the `Set` call actually resulted in a change, and only notify in those cases.